foundation-extensions/experience-platform-extension/release-notes.md

```diff
@@ -1,11 +1,23 @@
 # Release Notes
 
+## June 2, 2022
+
+### iOS AEPEdge 1.4.1
+
+* Updates the consent request to use "update" query operation in order to allow for incremental consent preferences changes.
+* Internal updates to use URLComponents builder for Edge endpoints.
+
+### Android Edge 1.3.2
+
+* Updates the consent request to use "update" query operation in order to allow for incremental consent preferences changes.
+* Internal updates to use ServiceProvider Network Service, reducing overall extension code size.
```

Comment:
Let's mention that it's the Mobile Core ServiceProvider to avoid confusion.
Something like "Updates internal network stack to use Mobile Core's ServiceProvider Network Service, reducing overall extension code size."